### PR TITLE
Add optional readOnly property to Input. close #60

### LINKED ISSUE #60: Add optional readOnly property to Input
I am using an Input and want it to be readOnly. since there is no optional readOnly property I am getting Lint warnings: 

 Warning: Failed form propType: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`. Check the render method of `Input`.

### PR DESCRIPTION
**Pull request by lbelinsk**

### CODE REVIEW

File: src/Input/Input.js [hunk 1]
```diff
@@ -132,7 +134,8 @@ class Input extends React.Component {
 Input.displayName = 'Input';
 
 Input.defaultProps = {
-  theme: 'normal'
+  theme: 'normal',
+  readOnly: false
```
Comment (mastertheblaster) [inline src/Input/Input.js]: No need defaults for the `false` value imho.

File: src/Input/README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 | defaultValue | string | - | - | Default value for those who wants to use this component un-controlled |
 | tabIndex  | number | - | - | Standard component tabIndex |
 | placeholder  | string | - | - | Placeholder to display |
+| readOnly  | bool | false | - | sets the input to readOnly |
```
Comment (mastertheblaster) [inline src/Input/README.md]: The same `Input` is being used by `AutoComplete` and `GoogleAutoComplete` components.
Would be very nice if this feature would provided for them as well.

File: src/Input/Input.spec.js [hunk 3]
```diff
@@ -40,6 +40,17 @@ describe('Input', () => {
     });
   });
 
+  describe('readOnly attribute', () => {
+    it('should pass down to the wrapped input', () => {
```
Comment (mastertheblaster) [inline src/Input/Input.spec.js]: Guess you should also check the default case? That by default it's not readonly.

Comment (lbelinsk) [inline src/Input/Input.spec.js]: I agree.. Will add this test

File: src/Input/Input.js (hunk 1 above)
Comment (lbelinsk) [inline src/Input/Input.js]: Thanks... I will update this...

File: src/Input/README.md (hunk 2 above)
Comment (lbelinsk) [inline src/Input/README.md]: I took a look in their code.
**AutoComplete** pass it automatically with the spreader operator {...props} to the Input.
**GoogleAddressInput** doesn't pass it to AutoComplete, so it requires a slight change in it.

Of course this prop needs to be documented in both of them. I will take care of that.

File: src/Input/Input.spec.js (hunk 3 above)
Comment (lbelinsk) [inline src/Input/Input.spec.js]: Done.

**Pull request merged**